setup.cfg

```diff
@@ -41,6 +41,7 @@ exclude = tests
 [options.extras_require]
 testing =
     pytest
+    pytest-asyncio
```

Comment:
Now `requests` can be removed from [setup.cfg](https://github.com/geraxe/dolib/pull/16/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52L32).